### DISCUSSION TITLE
Welcome to awesome-pt Discussions!

### DISCUSSION THREAD

**Discussion by sabersalv** [sabersalv/awesome-pt#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by enchained** [comment]
Hi. I've heard you're looking for maintainers? I'd like to volunteer, at least as a contributor. 

I love making detailed lists, categorizing them, etc. I would like to add more info about current tools, the difference between them, things like badges for used technology and repo activity. I also follow selfhosted/seedboxes/trackers on reddit and scripting threads on trackers to know about the latest automation tools. I could also fill website requests from issues and review pull requests. I think I'll be able to contribute to the project on my free time, and do a related tools research once every few months.

However, I do not speak Chinese and don't know much about Chinese trackers and tools, so this area might need a more experienced maintainer.

**Reply by sabersalv** [reply]
Hello @enchained 

Thank you for showing interest in this project. Please don't be concerned about the Chinese sections; I'm can assist you with those. I have added you as a collaborator—please feel free to reach out if you require any additional support.

Should you prefer a more immediate form of communication, feel free to leave your Telegram username, and we can continue our conversation there.
